src/main/java/edu/umich/its/cpm/MigrationController.java

```diff
@@ -207,7 +207,7 @@ private String login_becomeuser(HttpServletRequest request) {
 		// return the session id after login
 		String sessionId = "";
 
-		String remoteUser = request.getRemoteUser();
+		String remoteUser = "gsilver";
```

Comment:
The override remote user value could be a property.

On Mon, Dec 7, 2015 at 2:44 PM, Zhen Qian <notifications@github.com> wrote:

> In src/main/java/edu/umich/its/cpm/MigrationController.java
> <https://github.com/tl-its-umich-edu/ctools-project-migration/pull/56#discussion_r46866826>
> :
>
> > @@ -207,7 +207,7 @@ private String login_becomeuser(HttpServletRequest request) {
> >  		// return the session id after login
> >  		String sessionId = "";
> >
> > -		String remoteUser = request.getRemoteUser();
> > +		String remoteUser = "gsilver";
>
> You do not want to checkin the local overwrite of request.getRemoteUser()
> call.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tl-its-umich-edu/ctools-project-migration/pull/56/files#r46866826>
> .
>